### ISSUE TITLE
Annuals causing issues again

### ISSUE THREAD

**Issue by leaderdog** [mylar3/mylar3#449]
I have a bunch of annuals that just keep downloading and get stuck in retry.  I'm not sure if Comicvine changed how they handle them or if something in Mylar is to blame.

Annuals that I've already had for years are now not recognized by the title they are in (integrated annuals)

Guardians of the Galaxy 1990.  All 4 annuals are in the folder.  Yet Mylar cannot see them.

I tried refresh, I believe it gave an error.
I deleted the annuals and added them new into Mylar at the bottom of the page.
hit rescan
nothing. still sitting in wanted.

This seems to crop up now and again.  I've had this happened a few times.

Here is the care package.  I also added a jpg so you can see the folder, the Mylar Page and comicvine page.
https://www.dropbox.com/s/8b4oi3v1ly1tw3r/2020-09-12%20-%20carepackage.zip?dl=0

Python issue?  I'm not sure if Windows users can use anything higher than python 3.7 with Mylar yet?  if we can, I can upgrade it to 3.8.. I think ;)

Configuration Options
Mylar Version: python3-dev
-- git (build 9060fb19969e589f13a8931e5163dfa18407f1c1)
Python Version : 3.7.7
Language : en_CA.cp1252

**Comment by evilhero** [comment]
With the recent problems with the search results not accepting results with hyphens in them as a valid character, I had to change some things in order to fix that problem. 

In doing so, I changed the file parser to return an extra value, which it looks like when you're doing a file-check would cause an error. It actually stopped working on the first issue when you did a recheck files (or when it went to rescan the directory even I guess) - due to how you use the hyphen in your filenaming convention. So if you had taken any normal issue out of that given directory, and did a rescan it wouldn't have changed the status cause she gone done broken it.

Fix will be up in python3-dev in a bit...
